memory.go

```diff
@@ -86,15 +86,28 @@ func GetTopProcesses() {
 		return memoryPercentOfIthProcess > memoryPercentOfJthProcess
 	})
 
-	for i := 0; i < 5; i++ {
+	maxIndexAsStringLength := len(fmt.Sprintf("%d", howManyProcessesToShow))
+	topProcessesHeader := fmt.Sprintf("Top %d processes by memory usage:", howManyProcessesToShow)
+	ResultPrinter(topProcessesHeader, "")
+	for i := 0; i < howManyProcessesToShow; i++ {
 		memoryPercentOfIthProcess, err := processes[i].MemoryPercent()
 		if err != nil {
 			StandardPrinter(ErrorRedColor, "Process memory usage access denied.") //More descriptive error
 			panic(err)
 		}
```

Comment:
You need to handle out of range cases in this loop

Reply:
ok, thanks for testing

Reply:
I added:
```
howManyProcessesToShow := numberOfRequestedProcessesif len(processes) < howManyProcessesToShow { howManyProcessesToShow = len(processes) }
```